odoo_install.sh

```diff
@@ -250,6 +258,88 @@ sudo update-rc.d $OE_CONFIG defaults
 
 echo -e "* Starting Odoo Service"
 sudo su root -c "/etc/init.d/$OE_CONFIG start"
+#--------------------------------------------------
+# Install Nginx if needed
+#--------------------------------------------------
+if [ $INSTALL_NGINX = "True" ]; then
+  echo -e "\n---- Installing and setting up Nginx ----"
+  sudo apt install nginx -y
+  cat <<EOF > ~/$OE_USER
+   server {
+   listen 80;
+
+   # set proper server name after domain set
+   server_name $WEBSITE_NAME;
+
+   # Add Headers for odoo proxy mode
+   proxy_set_header X-Forwarded-Host \$host;
+   proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+   proxy_set_header X-Forwarded-Proto \$scheme;
+   proxy_set_header X-Real-IP \$remote_addr;
+   add_header X-Frame-Options "SAMEORIGIN";
+   add_header X-XSS-Protection "1; mode=block";
+
+   #   odoo    log files
+   access_log  /var/log/nginx/$OE_USER-access.log;
+   error_log       /var/log/nginx/$OE_USER-error.log;
+
+   #   increase    proxy   buffer  size
+   proxy_buffers   16  64k;
+   proxy_buffer_size   128k;
+
+   proxy_read_timeout 720s;
+   proxy_connect_timeout 720s;
+   proxy_send_timeout 720s;
+
+   #   force   timeouts    if  the backend dies
+   proxy_next_upstream error   timeout invalid_header  http_500    http_502
+   http_503;
+
+   types {
+   text/less less;
+   }
+
+   #   enable  data    compression
+   gzip    on;
+   gzip_min_length 1100;
+   gzip_buffers    4   32k;
+   gzip_types  text/css text/less text/plain text/xml application/xml application/json application/javascript application/pdf image/jpeg image/png;
+   gzip_vary   on;
+   client_header_buffer_size 4k;
+   large_client_header_buffers 4 64k;
+   client_max_body_size 0;
+
+   location / {
+   proxy_pass    http://127.0.0.1:$OE_PORT;
+   # by default, do not forward anything
+   proxy_redirect off;
+   }
+
+   location /longpolling {
+   proxy_pass http://127.0.0.1:8072;
```

Comment:
@tonybenoy make the `longpolling` configurable? So extra parameter at the top, which defaults to `8072`. This allows more flexibility on multi instance.